sdk/python/packages/flet/src/flet/cli/commands/pack.py

```diff
@@ -116,15 +116,36 @@ def add_arguments(self, parser: argparse.ArgumentParser) -> None:
         )
 
     def handle(self, options: argparse.Namespace) -> None:
+        is_dir_not_empty = lambda dir: os.path.isdir(dir) and len(os.listdir(dir)) != 0
+
         # delete "build" directory
         build_dir = os.path.join(os.getcwd(), "build")
-        if os.path.exists(build_dir):
-            shutil.rmtree(build_dir, ignore_errors=True)
+        if is_dir_not_empty(build_dir):
+            ask_to_delete = input('Do you want to delete "build" directory? (y/n) ')
```

Comment:
Does `build` directory have any value to care about its deletion? A developer won't be doing/customizing there anything?

Reply:
Just in case, the user has their some important files in `build` directory. I would remove this if you want it to be removed by the way,,